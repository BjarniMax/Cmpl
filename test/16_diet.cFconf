-osil 16_diet.osil
-osrl 16_diet.osrl
-serviceMethod solve
