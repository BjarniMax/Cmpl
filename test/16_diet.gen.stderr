error (input/output): Input file '16_diet.gen' not found

